tests/postgresql_test.go

```diff
@@ -106,7 +109,10 @@ func TestPgReadReplica(t *testing.T) {
 	assert.Equal(t, replicaAvn.State, replica.Status.State)
 	assert.Equal(t, replicaAvn.Plan, replica.Spec.Plan)
 	assert.Equal(t, replicaAvn.CloudName, replica.Spec.CloudName)
-	assert.Equal(t, map[string]string{"env": "test", "instance": "replica"}, replica.Spec.Tags)
+	assert.Equal(t, map[string]string{"env": "test", "instance": "foo"}, replica.Spec.Tags)
```

Comment:
Hmmm. I was too hasty.

Reply:
Oh, oops. I did that change with a search-replace, guess I didn't check the diff well enough.

But I think the test passed ... 🤔